api/overdrive.py

```diff
@@ -1006,13 +1007,13 @@ def patron_request(self, patron, pin, *args, **kwargs):
         return response
 
 
-class OverdriveCirculationMonitor(CollectionMonitor):
+class OverdriveCirculationMonitor(CollectionMonitor, TimelineMonitor):
     """Maintain LicensePools for recently changed Overdrive titles. Create
     basic Editions for any new LicensePools that show up.
     """
     SERVICE_NAME = "Overdrive Circulation Monitor"
-    INTERVAL_SECONDS = 500
     PROTOCOL = ExternalIntegration.OVERDRIVE
+    OVERLAP = datetime.timedelta(minutes=1)
```

Comment:
This fits the `TimelineMonitor` mold, and we previously didn't use any overlap at all, so I set the OVERLAP value to something smaller than five minutes.

I do think having a little overlap will be good here; we sometimes miss events from Overdrive and it's possible they're falling into the cracks in between runs.